docs/learn/master-data-hub/configuration-examples.md

```diff
@@ -128,6 +128,78 @@ Compound queries are very useful when we need to match against multiple attribut
 }
 ```
 
+## Fuzzy match score normalization
+
+Score returned normalized to interval between 0-1.
+
+```json
+{
```

Comment:
Consider wrapping it into the `"aggregate"` array to be consistent with other examples (as a followup).